README.md

```diff
@@ -78,21 +77,22 @@ Now that you have more data for each autocomplete item, you'll want to customize
 The default rendering function is shown below:
 
 	var defaultRenderFunction = function(document_type, item) {
-		return '<p class="title">' + item['title'] + '</p>';
+		return '<p class="title">' + item.highlight['title'] + '</p>';
 	};
 
 The additional fields are available as keys in the item dictionary, so you could customize this to make use of the `genre` field as follows:
 
 	var customRenderFunction = function(document_type, item) {
-		var out = '<p class="title">' + item['title'] + '</p>';
-		return out.concat('<p class="genre">' + item['genre'] + '</p>');
+	  var out = '<a href="' + item['url'] + '" class="st-search-result-link">' + item.highlight['title'] + '</a>';
```

Comment:
Ah, actually, here's a perfect example. Using `item['url']` here IS still actually an XSS vulnerability.

So you could change to:
`Swiftype.htmlEscape(item['url'])`